model/src/main/java/oracle/kubernetes/weblogic/domain/ServerConfigurator.java

```diff
@@ -4,10 +4,20 @@
 
 package oracle.kubernetes.weblogic.domain;
 
+/** An interface for an object to configure a server in a test. */
+@SuppressWarnings("UnusedReturnValue")
```

Comment:
Just curious...  I've seen you add lots of these statements to suppress warnings, which I'm guessing are specific to IntelliJ.  Since I use Eclipse (and the team is mixed), I have no way to know which warnings I should suppress, or no longer need to suppress.  Do you know of any standard way to configure the IDE's to respect the same set of warnings?

Reply:
Unfortunately, the Java team hasn't published a standard list of warnings, as far as I know. In Intellij, I can suppress a warning to acknowledge that I am aware of a problem and consider it acceptable. I do that, though, by clicking on the warning and asking the editor to add the annotation.

I think that Eclipse often uses the same annotations, but not always, which is really annoying. I suspect it should be OK to add both, when they use different ones, but I don't know how to do that in Eclipse.